Userland/Libraries/LibGfx/ImageFormats/ILBMLoader.cpp

```diff
@@ -96,13 +106,14 @@ static ErrorOr<void> decode_iff_ilbm_header(ILBMLoadingContext& context)
     return {};
 }
 
-static ErrorOr<FixedArray<Color>> decode_cmap_chunk(Chunk cmap_chunk)
+static ErrorOr<Vector<Color>> decode_cmap_chunk(Chunk cmap_chunk)
 {
     size_t const size = cmap_chunk.data.size() / 3;
-    FixedArray<Color> color_table = TRY(FixedArray<Color>::create(size));
+    Vector<Color> color_table;
+    color_table.ensure_capacity(size);
```

Comment:
We are becoming less strict about allocation failures in userland, but image decoders are based on untrusted data, so we should use `try_ensure_capacity` here.